docker-setup/README.md

```diff
@@ -0,0 +1,2 @@
+# Snowflake Sink Connector Docker 
+Onboarding instructions here: [Snowflake Kafka Connector In Docker Container](https://snowflakecomputing.atlassian.net/wiki/spaces/EN/pages/2466252387/Snowflake+Kafka+Connector+In+Docker+Container)
```

Comment:
do you want to update the doc as well to reflect the process changes after this is merged?

Reply:
The dock has already been updated! 
https://snowflakecomputing.atlassian.net/wiki/spaces/EN/pages/2466252387/Snowflake+Kafka+Connector+In+Docker+Container